### PR TITLE
Pojos migration

### PR DESCRIPTION
**Pull request by jburel**
Update ref to pojo
See https://github.com/openmicroscopy/openmicroscopy/pull/4128

### CODE REVIEW

File: omero/developers/Modules/Api.txt [hunk 1]
```diff
@@ -159,15 +159,15 @@ defines all the actions necessary to administer the
...
+Model Object Java
+^^^^^^^^^^^^^^^^^
 
 Certain operations, like those dealing with data management and viewing,
 happen more frequently than others (like defining microscopes). Those have
 been collected in the 
 :source:`IContainer <components/common/src/ome/api/IContainer.java>`
 interface. IContainer simplifies a few very common queries, and there is a
-related package ("pojos.\*") for working with the returned graphs.
+related package ("omero.gateway.model.\*") for working with the returned graphs.
```
Comment (sbesson) [inline omero/developers/Modules/Api.txt]: To simplify further maintenance, I would be tempted to link at the API Javadoc here, i.e.
http://downloads.openmicroscopy.org/omero/5.1.3/api/pojos/package-summary.html

**Comment by jburel** [comment]
@sbesson: not planning another migration ;-)

**Comment by hflynn** [comment]
Code PR is merged now I see, is this waiting on anything?
There are still a couple of gateway broken links on https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-latest-docs/18/warnings3Result/ but I assume they are related to versioning

**Comment by sbesson** [comment]
@hflynn: correct. Broken links in the latest build not showing up in the merge build are related to the state of the code at the latest tag (here 5.2.0-m0) and will be fixed once we create an intermediate milestone release.

**Comment by hflynn** [comment]
I figured as much. So does this PR still need some more work or is it good to go?

**Comment by jburel** [comment]
not planning more changes unless we want to implement @sbesson suggestion now and he will do the next pojos migration ;-)

**Comment by hflynn** [comment]
I'll merge this for now then, shall I?

**Comment by hflynn** [comment]
--no-rebase

**Pull request merged**